task/github.py

```diff
@@ -321,19 +324,20 @@ def statuses(self, revision):
                 count = len(data["statuses"])
         return result
 
-    def all_statuses(self, revision):
+    def all_statuses(self, revision: str) -> List[Any]:
```

Comment:
`List[Any]` isn't any better than just writing `list` as far as I know.  And both of them aren't really good enough...